discovery_engine_core/web-api/src/lib.rs

```diff
@@ -19,6 +19,9 @@ mod routes;
 mod state;
 mod storage;
 
+#[macro_use]
+extern crate lazy_static;
```

Comment:
:wrench: from rust edition 2018 onwards `#[macro_use]` and `extern crate` shouldn't be necessary anymore. you can directly import the macro via `use lazy_static::lazy_static;` in the module where it's needed (not necessarily at `crate` level).

Reply:
Okay, I went with the tutorial on this one, must have been obsolete. I will change it

Reply:
:art: Using `once_cell` is a bit nicer (nice enough for it to be in nightly std).

I.e.:

```
use once_cell::sync::Lazy;
static RE: Lazy<Regex> = Lazy::new(||  .... );
```